spec/rails_helper.rb

```diff
@@ -35,7 +35,7 @@
 end
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_path = Rails.root.join('/spec/fixtures').to_s
```

Comment:
Itt szerintem nem lesz szükség a `to_s` hívásra. Légyszi ellenőrizd le, hogy `to_s` nélkül is lefutnak-e a tesztek!